net-p2p/classified-ads/classified-ads-0.13.ebuild

```diff
@@ -0,0 +1,87 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+PLOCALES="en fi sv da uk es"
+PLOCALE_BACKUP="en"
+inherit qmake-utils virtualx l10n
```

Comment:
You're not using l10n at all here (and you aren't supposed to), so please remove that and `PLOCALE*`.